Initial documentation on how to setup the demo (#10)
anbox-cloud/cloud-gaming-demo: README.md

# Overview

> This project demonstrates a simple cloud gaming service on top of [Anbox Cloud](https://anbox-cloud.io)
> allowing users to play Android based games right from their web browser on any device.

Reviewer: ```suggestion
This project demonstrates a simple cloud gaming service on top of [Anbox Cloud](https://anbox-cloud.io)
that allows users to play Android-based games right from their web browser on any device.
```

The demo is build of two components:

* A web UI based on[Flutter](https://flutter.dev/)
* A backend service written in [Python](https://www.python.org/)

> The backend service will connect to the Anbox Cloud services and retrieve a list of available applications
> and also take care of launching actual streaming session on demand.

Reviewer: ```suggestion
The backend service connects to the Anbox Cloud services and retrieves a list of available applications.
It also takes care of launching the actual streaming sessions on demand.
```

> Inside the web UI the user can select from the list of available games and starting playing each of them.

Reviewer: ```suggestion
Inside the web UI, the user can choose from the list of available games and start playing either of them.
```

> # Quickstart

Reviewer: ```suggestion
## Quick start
```

> Running the demo yourself is as simple as running through the following steps:
> 
> 1. Install the Anbox Cloud Appliance. You can find more details [here](https://anbox-cloud.io/docs/tut/installing-appliance)
> 2. SSH to your appliance instance, clone the demo repository and run the installation script:

Reviewer: ```suggestion
To run the demo, simply complete the following steps:

1. Install the Anbox Cloud Appliance. See the [documentation](https://anbox-cloud.io/docs/tut/installing-appliance) for instructions.
2. SSH to your appliance instance, clone the demo repository and run the installation script:
```

> ```

Reviewer: I think this must be indented with three spaces to display correctly - but GitHub doesn't let me suggest that. ;)

> $ git clone git://github.com/anbox-cloud/cloud-gaming-demo
> $ cd cloud-gaming-demo
> $ sudo scripts/install.sh

Reviewer: ```suggestion
   git clone git://github.com/anbox-cloud/cloud-gaming-demo
   cd cloud-gaming-demo
   sudo scripts/install.sh
```

```

The installation will take a moment as it will fetch and install a selection of freely available Android
game titles.

At the end the installation script will print the URL you can access the demo on, happy gaming!

> # Get Help & Community
> 
> If you get stuck deploying the demo or would like to have help with Anbox Cloud in general, come and ask
> over on the [Anbox Cloud discourse forum](https://discourse.ubuntu.com/c/anbox-cloud)!

Reviewer: ```suggestion
## Get help & community

If you get stuck deploying the demo or would like some help with Anbox Cloud in general, come and ask on the [Anbox Cloud discourse forum](https://discourse.ubuntu.com/c/anbox-cloud)!
```

> # More information
> 
> You can find more information about Anbox Cloud on its [website](https://anbox-cloud.io) and in its
> [documentation](https://anbox-cloud.io/docs).
> 
> If you're interested in building out a real production cloud gaming service, [come an talk to us](https://anbox-cloud.io/contact-us)!

Reviewer: ```suggestion
## More information

You can find more information about Anbox Cloud on the [website](https://anbox-cloud.io) and in the
[documentation](https://anbox-cloud.io/docs).

If you're interested in building a real production cloud gaming service, [get in touch](https://anbox-cloud.io/contact-us)!
```
